x-pack/plugins/reporting/export_types/csv/server/lib/flatten_hit.js

```diff
@@ -6,6 +6,7 @@
 
 import _ from 'lodash';
 
+// FIXME shouldn't this really use a service provided by @kbn-es-query? The logic here can part from Discover
```

Comment:
Nitpick: `The logic here can part from Discover`, for some reason I have a hard time reading this. Does this mean that the logic here can be separated from Discover? IE right now it's closely coupled to it?

Reply:
I'll fix the comment, but this is meaning that the logic can part ways with Discover, even though they do much of the same thing. That becomes the cause of bugs like https://github.com/elastic/kibana/issues/25068